public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -136,9 +136,7 @@ export const getDefaultQueries = (): GrafanaQuery[] => {
   if (!dataSource) {
     return [getDefaultExpression('A')];
   }
-
-  const timeRange = getDefaultTimeRange();
-  const relativeTimeRange = rangeUtil.timeRangeToRelative(timeRange);
+  const relativeTimeRange = { from: 600, to: 0 };
```

Comment:
I think there is an `getDefaultRelativeTimeRange` function. Maybe something we can use here?